### DISCUSSION TITLE
Using the sequence flags of the SPP primary header

### DISCUSSION THREAD

**Discussion by guidoSNT** [yamcs/yamcs#946] [Q&A]
Hi! I've been trying to use the sequence flag headers of the Space Packet Protocol (SPP), and even though the flag can be read, I don't know if it's possible to use it to reconstruct larger messages. For example, we have 96-byte packets that can send up to 6kB messages using these flags, but we can't read them in Yamcs as a single message.

So, is there any way we can declare that when some packets are received in a certain sequence, they should be interpreted as one?

**Comment by xpromache** [comment]
I think you can build that functionality in a custom pre-processor which you may need to make anyway for your mission.
